### DISCUSSION TITLE
Offline pyodide.js - local and not from jsdeliver

### DISCUSSION THREAD

**Discussion by sklibanov** [flet-dev/flet#2264] [Q&A] [answered]
### Question

So I'm using flet publish (https://flet.dev/docs/guides/python/publishing-static-website/) to make a static website. I want that website to serve *everything*, including all the packages and js that's going to run inside the browser.

At the top of client/web/python-worker.js however, we have this:

```
importScripts("https://cdn.jsdelivr.net/pyodide/v0.24.1/full/pyodide.js");
```

so it's doing an https fetch from there, not from my local server.

So, I have these questions:

1. What other dependencies do we load from the Internet that are not packaged at ```flet publish``` time?
2. Is it reasonable to attempt local changes to flet to add support for local copies of everything? Or is it an impossible or misguided task?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment, marked as answer]
We'll implement bundling of pyodide with a Flet app.

**Reply by FeodorFitsner** [reply]
Issue to track: https://github.com/flet-dev/flet/issues/2276
